src/rest/src/main/java/org/geoserver/rest/converters/StyleWriterConverter.java

```diff
@@ -40,7 +40,7 @@ public StyleWriterConverter(String mimeType, Version version, StyleHandler handl
 
     @Override
     protected boolean supports(Class<?> clazz) {
-        return Style.class.isAssignableFrom(clazz) || StyleInfo.class.isAssignableFrom(clazz);
+        return RestWrapper.class.isAssignableFrom(clazz) || Style.class.isAssignableFrom(clazz) || StyleInfo.class.isAssignableFrom(clazz);
```

Comment:
This writer was only intended to write out style contents. I believe adding RestWrapper here will cause it to also try writing StyleInfo objects as well (which should be handled by the standard XStream writers instead).